realsense2_camera/launch/rs_launch.py

```diff
@@ -22,6 +22,7 @@
 
 
 configurable_parameters = [{'name': 'camera_name',                  'default': 'camera', 'description': 'camera unique name'},
+                           {'name': 'namespace',                    'default': '', 'description': 'namespace for camera'},
```

Comment:
I don't want to surprise current users if they pull from this branch and build again.
Most of them used to see `/camera/camera` when launching our node, so I would prefer to set the default value in the namespace to `"camera"`, to keep this behavior.

Reply:
Also, I think it is better to name this parameter `camera_namespace` to make it more clean for users that it defines a namespace for the camera.

Reply:
sure let's go with `camera_namespace`